DQM/EcalMonitorClient/src/LedClient.cc

```diff
@@ -165,6 +165,10 @@ namespace ecaldqm {
 
         float aRmsThr(sqrt(pow(aMean * toleranceAmpRMSRatio_, 2) + pow(3., 2)));
```

Comment:
This variable seems to be unused.

Reply:
Yes, but if that's commented out it will complain about other variables like `toleranceAmpRMSRatio_`, which would need to be commented out here as well as the header and .py files. And since this is supposed to be a temporary disabling of the cuts and would need to be reused in the future, I left it untouched to keep the changes to a minimum.